modules/apps/commerce/commerce-currency-service/src/main/java/com/liferay/commerce/currency/internal/search/spi/model/query/contributor/CommerceCurrencyModelPreFilterContributor.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.currency.internal.search.spi.model.query.contributor;
+
+import com.liferay.commerce.product.constants.CPField;
+import com.liferay.portal.kernel.search.SearchContext;
+import com.liferay.portal.kernel.search.filter.BooleanFilter;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.search.spi.model.query.contributor.ModelPreFilterContributor;
+import com.liferay.portal.search.spi.model.registrar.ModelSearchSettings;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Mahmoud Azzam
+ */
+@Component(
+	property = "indexer.class.name=com.liferay.commerce.currency.model.CommerceCurrency",
+	service = ModelPreFilterContributor.class
+)
+public class CommerceCurrencyModelPreFilterContributor
+	implements ModelPreFilterContributor {
+
+	@Override
+	public void contribute(
+		BooleanFilter booleanFilter, ModelSearchSettings modelSearchSettings,
+		SearchContext searchContext) {
+
+		_filterByActive(booleanFilter, searchContext);
+	}
+
+	private void _filterByActive(
+		BooleanFilter booleanFilter, SearchContext searchContext) {
+
+		Boolean active = (Boolean)GetterUtil.getObject(
+			searchContext.getAttribute(CPField.ACTIVE));
+
+		if (active != null) {
```

Comment:
this will never be null with the GetterUtil, I think you should get the attribut out of the searchContext without the GetterUtil

Reply:
GetterUtil has the default value set to null